apps/src/lib/ui/lti/sync/LtiSectionSyncDialogHelpers.tsx

```diff
@@ -1,21 +1,12 @@
 import i18n from '@cdo/locale';
 
-export const getRosterSyncErrorMessage = (errorCode: number) => {
-  switch (errorCode) {
-    case 400:
-      return i18n.ltiSectionSyncDialogErrorWrongContext();
-    case 401:
+export const getRosterSyncErrorMessage = (error: string) => {
+  switch (error) {
```

Comment:
Since we're using a switch statement here, we get the benefit of falling through to a default case; might as well take advantage of that and include a generic default case.